app/views/content/life-as-a-teacher/explore-subjects/languages.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Become a languages teacher
+title_paragraph: |-
+  <p>
+  Learning a language teaches pupils more than just grammar and vocabulary. As a languages teacher, you'll help pupils gain a greater appreciation and understanding of the world.</p>
+  <p>
```

Comment:
Still need to change this header @Sarah-DfE - we can't list figures explicitly like this and the line below without a caveat whilst we are in this interim period

Suggest changing to `Tax-free bursaries are available for eligible trainee language teachers.` as Emma has done on other subject pages

(I appreciate this doesn't mention scholarships but that might be the easiest thing for now - they are referenced further down the page.)